config/config.toml

```diff
@@ -147,4 +147,9 @@ peer_seeds = [
 committee = ["2ea0df3059caf4411624d6bf5b9c02238d607d2798c586b3e6c2a054da3f205a"] # cannot be of zero size
 phase_timeout = 30
 template_id = "EditableMetadata"
-
+[collectibles]
```

Comment:
I think this file has been unintentionally included in source some how and should be deleted